src/index.js

```diff
@@ -20,23 +21,40 @@ import output from './output';
  * This is adapter function that will send the data to the chosen output module
  * @param {*} data 
  */
-const sendToOutput = (...data) => {
+const sendToOutput = (data) => {
   output(data);
 };
 
 /**
  * This function will decode the transaction and the logs that happed inside it,
  * then send them to the out put function 
  * @param {*} transaction 
- * @param {*} logs 
  */
-const transactionHandler = (transaction) => {
-  const decodedInputDataResult = decodeInputData(transaction); // eslint-disable-line
+const transactionHandler = async (transaction) => {
+  /**
+   * Load the ABI object from the memeory
```

Comment:
`Load ABI object from the memory` i think there is no need for this comment but we can keep if you think its crucial